Trying to make it work for ROS iron and elixir 1.15.6-otp-26

I'm trying to run the package with the versions specified above, however on a clean install after running `mix deps.compile` I get the following error

`mix deps.compile
==> rclex
cc -o /home/ed/projects/ros_elixir_ws/src/hello-elixir/src/talker/_build/dev/lib/rclex/obj/graph_nif.o -c -g -O2 -Wall -Wextra -Wno-unused-parameter -pedantic -fPIC -I./src -I/home/ed/.asdf/installs/erlang/26.1.1/usr/include -I/opt/ros/iron/include/rcl -I/opt/ros/iron/include/rcl_yaml_param_parser -I/opt/ros/iron/include/rcutils -I/opt/ros/iron/include/rmw -I/opt/ros/iron/include/rosidl_runtime_c -I/opt/ros/iron/include/rosidl_typesupport_interface -I/opt/ros/iron/include/rosidl_dynamic_typesupport -I/opt/ros/iron/include/actionlib_msgs -I/opt/ros/iron/include/action_msgs -I/opt/ros/iron/include/diagnostic_msgs -I/opt/ros/iron/include/geometry_msgs -I/opt/ros/iron/include/lifecycle_msgs -I/opt/ros/iron/include/map_msgs -I/opt/ros/iron/include/nav_msgs -I/opt/ros/iron/include/pcl_msgs -I/opt/ros/iron/include/pendulum_msgs -I/opt/ros/iron/include/rosgraph_msgs -I/opt/ros/iron/include/sensor_msgs -I/opt/ros/iron/include/service_msgs -I/opt/ros/iron/include/shape_msgs -I/opt/ros/iron/include/statistics_msgs -I/opt/ros/iron/include/std_msgs -I/opt/ros/iron/include/stereo_msgs -I/opt/ros/iron/include/tf2_geometry_msgs -I/opt/ros/iron/include/tf2_msgs -I/opt/ros/iron/include/tf2_sensor_msgs -I/opt/ros/iron/include/trajectory_msgs -I/opt/ros/iron/include/unique_identifier_msgs -I/opt/ros/iron/include/visualization_msgs src/graph_nif.c
In file included from /opt/ros/iron/include/rcl/rcl/rcl.h:77,
                 from src/graph_nif.c:11:
/opt/ros/iron/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:78: /home/ed/projects/ros_elixir_ws/src/hello-elixir/src/talker/_build/dev/lib/rclex/obj/graph_nif.o] Error 1
could not compile dependency :rclex, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile rclex --force", update it with "mix deps.update rclex" or clean it with "mix deps.clean rclex"
==> 
** (Mix) Could not compile with "make" (exit status: 2).
You need to have gcc and make installed. If you are using
Ubuntu or any other Debian-based system, install the packages
"build-essential". Also install "erlang-dev" package if not
included in your Erlang/OTP version. If you're on Fedora, run
"dnf group install 'Development Tools'".`

erlang-dev and build-essential is already installed.

Would like to know any guidance on how to get it working

--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ ifeq ($(ROS_DISTRO), humble)
 ROS_INCS    ?= rcl rcl_yaml_param_parser rcutils rmw rosidl_runtime_c rosidl_typesupport_interface
 ROS_CFLAGS  ?= $(addprefix -I$(ROS_DIR)/include/, $(ROS_INCS)) $(addprefix -I, $(wildcard $(ROS_DIR)/include/*_msgs))
 else ifeq ($(ROS_DISTRO), iron)
-ROS_INCS    ?= rcl rcl_yaml_param_parser rcutils rmw rosidl_runtime_c rosidl_typesupport_interface rosidl_dynamic_typesupport
+ROS_INCS    ?= rcl rcl_yaml_param_parser rcutils rmw rosidl_runtime_c rosidl_typesupport_interface rosidl_dynamic_typesupport type_description_interfaces builtin_interfaces
 ROS_CFLAGS  ?= $(addprefix -I$(ROS_DIR)/include/, $(ROS_INCS)) $(addprefix -I, $(wildcard $(ROS_DIR)/include/*_msgs))
 else
 ROS_CFLAGS  ?= -I$(ROS_DIR)/include

